src/sign-in-blazorwasm/README.md

```diff
@@ -109,16 +80,15 @@ Use the following settings for your app registration:
 
 ### 2. Signin into the web app
 
-1. Once the web app is listening, navigate to https://localhost:5001
-1. Sign-in with your user credentials.
+1. Once the web app is listening, navigate to https://localhost:5001 and enter tour crendentials
 
-### 3. Clean up
+![A screenshot of an ASP.NET Core 6.0 Blazor WebAssembly application displaying a response from Microsoft Graph.](./app-signedin.png)
 
-1. Delete the Azure AD app
+1. Click Logout
 
-   ```bash
-   az ad app delete --id $AZURE_AD_APP_CLIENT_ID_BLAZORWASM
-   ```
+![A screenshot of an ASP.NET Core 6.0 Blazor WebAssembly application indicating the user signed-out and allowing click "Login" to signin again.](./app-signedout.png)
+
+### 3. Clean up
```

Comment:
What is being cleaned up?

Reply:
good catch @ckittel 

done | addressed from [bd62e](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/21/commits/bd62e12bc660a4a7b406a8c8dd8f252eee91af52)